src/DetailsView/components/auto-detected-failures-dialog.tsx

```diff
@@ -39,7 +53,14 @@ export class AutoDetectedFailuresDialog extends React.Component<
             Object.entries(this.props.visualizationScanResultData.tabStops.requirements).some(
                 ([_, data]) => data.instances.length > 0 && data.status === 'fail',
             );
-        if (tabbingJustFinished && autoDetectedFailuresExist && !prevState.dialogEnabled) {
+        const dialogEnabled = this.props.userConfigurationStoreData.showAutoDetectedFailuresDialog;
```

Comment:
nit: the naming here is a bit off since it collides with the `dialogEnabled` state variable, which refers to something else. I think either renaming this to `showAgainSetting` (or similar) OR to rename the other state variable to something else might be best.

Reply:
+1 on this